packages/muelu/test/scaling/MatrixLoad.hpp

```diff
@@ -28,8 +28,38 @@
 #include <Galeri_XpetraUtils.hpp>
 #include <Galeri_XpetraMaps.hpp>
 
+#include "MueLu_Zoltan2Interface_decl.hpp"
```

Comment:
Zoltan2 is an optional dependency. Is this guaranteed to work? Also, why "_decl.hpp" instead of ".hpp"?

Reply:
Changed to follow pattern in, e.g., `MueLu_Factory_def.hpp` -- `MueLu_Zoltan2Interface.hpp` is OK to include unconditionally, but the repartitioning utilities need to be guarded against by a:

```cpp
#if defined(HAVE_MUELU_ZOLTAN2) && defined(HAVE_MPI)
// distribute using Zoltan
#else
// no-op
#endif